constraints/cpu.md

```diff
@@ -0,0 +1,101 @@
+# CPU Constraints in Kata Containers
+
+## Default number of virtual CPUs
+
+Before starting a container, the [runtime][6] reads the `default_vcpus` option from the [configuration file][7] to determine the number of virtual CPUs (vCPUs) needed to start the virtual machine. By default, `default_vcpus` is equal to 1 for fast boot time and a small memory footprint per virtual machine. Be aware that increasing this value negatively impacts the virtual machine's boot time and memory footprint.
```

Comment:
Could you break down all paragraphs on this document into single lines ? I'm not 100% sure, but I think that's how all our docs are written. This way, it is easier to make some comments for each line if needed.

Reply:
`this value` is correct since I'm talking about `default_vcpus`